lib/Installation/ModuleSelection/ModuleSelectionPage.pm

```diff
@@ -39,22 +39,29 @@ sub is_shown {
     return $self->{rt_items}->exist();
 }
 
-sub get_modules {
+sub get_modules_full_name {
     my ($self) = @_;
     my @modules = ($self->{rt_items}->text() =~ /<a href='(.*?)'.*<\/a>/g);
```

Comment:
and the full name for text mode? I think it is only needed as I mentioned before for selecting something in textmode which seems that we don't do. But we can leave it with a comment.wdyt?